os/arch/arm/src/artik053/src/artik053_tash.c

```diff
@@ -179,21 +179,20 @@ static void artik053_configure_partitions(void)
 		} else
 #endif
 #if defined(CONFIG_MTD_SMART) && defined(CONFIG_FS_SMARTFS)
-			if (!strncmp(types, "smartfs,", 8)) {
-				char partref[4];
-				snprintf(partref, sizeof(partref), "p%d", partno);
-				smart_initialize(CONFIG_ARTIK053_FLASH_MINOR, mtd_part, partref);
-			} else
+		if (!strncmp(types, "smartfs,", 8)) {
```

Comment:
Don't mix formatting like an indentation and code changes in a commit.
Please revert or split this file changes.